adam/curriculum_to_html.py

```diff
@@ -128,48 +128,33 @@ def _dump_instance_group(
             html_out.write(f"<head>\n\t<style>{CSS}\n\t</style>\n</head>")
             html_out.write(f"\n<body>\n\t<h1>{title} - {instance_group.name()}</h1>")
             html_out.write("\n<h2>How to read</h2>")
+            html_out.write(f"\n<p>Our curiculum is designed in two levels: The objective, here - {instance_group.name()}")
             html_out.write(
-                "\n<p>A Situation is curriculum-designer-facing; it is not accessible to the learner. "
-                "The name of a perceived object is derived from the type of the corresponding SituationObject, and the "
-                "number indicates which object of its type it is - e.g. if there are two babies in a situation, "
-                "the perceived objects will be 'person_0' and 'person_1'. "
-                "Names on Perceptions are handles for debugging; they are not accessible to the "
-                "learner. 'GAZED-AT' refers the object of focus of the speaker.</p> "
+                "\n and a set of scenes for each objective. The scenes are in the shaded table below."
```

Comment:
Something seems to be missing? This starts with "and a set of scenes..."?

Reply:
i suspect this is bad merging--- should I edit in the file?

Reply:
actually, not bad merging... perhaps bad use of python. 
```
            html_out.write(f"\n<p>Our curiculum is designed in two levels: The objective, here - {instance_group.name()}")
            html_out.write(
                "\n<p>Our curiculum is designed in two levels: The objective, here - {instance_group.name},"
                "\n and a set of scenes for each objective. The scenes are in the shaded table below."
                "\n For each scene, we display: "
                "\n\t<li>Situation (column 1), which is used by the curriculum developer to specify key objects and actions,"
                "\n but is unseen by the learner</li>."  
                "\n\t<li>Language (column 2), the text presented to the learner</li>."  
                "\n\t<li>Learner Perception (column 3), a textual representaiton of the perceptual features"
                "\n presented to the learner</li>"             )
                "\n presented to the learner</li>")
```

Reply:
and the repeated line I had thought was deleted.